modules/database/sqlite/site-health.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * Adds and filters data in the site-health screen.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Filter debug data in site-health screen.
+ *
+ * When the plugin gets merged in wp-core, these should be merged in src/wp-admin/includes/class-wp-debug-data.php
+ * See https://github.com/WordPress/wordpress-develop/pull/3220/files
+ *
+ * @since n.e.x.t
+ *
+ * @param array $info The debug data.
+ */
+function perflab_sqlite_plugin_filter_debug_data( $info ) {
+	$database_type = defined( 'DATABASE_TYPE' ) && 'sqlite' === DATABASE_TYPE ? 'sqlite' : 'mysql';
+	if ( 'sqlite' !== $database_type ) {
+		return $info;
+	}
+
+	$info['wp-constants']['fields']['DATABASE_TYPE'] = array(
+		'label' => 'DATABASE_TYPE',
+		'value' => ( defined( 'DATABASE_TYPE' ) ? DATABASE_TYPE : __( 'Undefined', 'performance-lab' ) ),
+		'debug' => ( defined( 'DATABASE_TYPE' ) ? DATABASE_TYPE : 'undefined' ),
+	);
+
+	$info['wp-database']['fields']['database_type'] = array(
+		'label' => __( 'Database type', 'performance-lab' ),
+		'value' => 'sqlite' === $database_type ? 'SQLite' : 'MySQL/MariaDB',
```

Comment:
`'sqlite' === $database_type` is always true, becase it was already checked in line 21.

Reply:
I was writing these with the intent to make it as easy as possible to then copy/paste to Core with minimal tweaks - that's why I included these extra conditions. When back ported to Core, this condition will need to be here 😉 

Of course I can remove it for the sake of the plugin, but we'll just have to write them again when backporting

Reply:
@aristath It makes sense to put condition over here if we are going to directly port this module as is into core. But yes, we can remove the condition from here, and put a note here saying when porting to core, should consider putting this condition back. Best judgement will be yours in this case. 👍

Reply:
I actually found an edge-case issue with the site-health screen and pushed a fix for it - which required these checks to be there. The problem was that when the module is active but the `wp-content/db.php` file is missing, the database section was reporting `SQLite` as the site's db instead of `MySQL`.
Fixing that required moving around some things, and ensuring that when MySQL is the active db, data is still reported correctly 👍